http-service/src/main/java/net/runelite/http/service/hiscore/HiscoreService.java

```diff
@@ -126,13 +182,24 @@ public SingleHiscoreSkillResult singleSkillLookup(@PathVariable String skillName
 		return skillResult;
 	}
 
-	public HttpUrl getUrl()
+	@InitBinder
+	public void initBinder(WebDataBinder binder)
+	{
+		binder.registerCustomEditor(HiscoreEndpoint.class, new HiscoreEndpointEditor());
+	}
+
+	public void setUrl(HiscoreEndpoint endpoint)
+	{
+
+	}
+
+	public HttpUrl getTestUrl()
 	{
-		return url;
```

Comment:
I think introducing "testUrl"s here is a code smell. I think this should be revisited.